_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -296,6 +296,55 @@ Use the following steps to register a snapshot repository backed by an Azure sto
    }
    ```
 
+### General set up to use Microsoft Azure Blob Storage
+
+To utilize Microsoft Azure Blob Storage as a snapshot repository, follow the steps below:
+1. Install the `repository-azure` plugin on all nodes with the following command:
+
+   ```bash
+   ./bin/opensearch-plugin install repository-azure
+   ```
+
+1. Once the repository-azure plugin is installed, define your Azure Blob Storage settings before initializing the node. Start by defining your Azure Storage account name using the following secure setting:
+
+   ```bash
+   ./bin/opensearch-keystore add azure.client.default.account
+   ```
+
+1. choose one of the following options to set up your authentication credentials for Azure Blob Storage:
+
+   #### Using Azure storage account key
+   
+   Use the following setting to specify your Azure Storage account key:
+   
+   ```bash 
+   ./bin/opensearch-keystore add azure.client.default.key
+   ```
+
+   #### Shared access signature (SAS)
+   
+   Use the following setting when accessing Azure with a shared access signature:
+         
+   ```bash
+   ./bin/opensearch-keystore add azure.client.default.sas_token      
+   ```
+
+   #### Azure token credential 
+
+   Starting from OpenSearch 2.15.0, you have the option to configure a token credential authentication flow in `opensearch.yml`. This method is distinct from connection string authentication, which necessitates a SAS or an account key.
+
+   If you opt for token credential authentication, you will need to choose a token credential type. Although Azure offers multiple token credential types, as of OpenSearch version 2.15.0, only managed identity is supported.
+
+   To use managed identity, add your token credential type to `opensearch.yml` using either `managed` or `managed_identity` values. This indicates that managed identity is being used to perform token credential authentication:
+
+   ```yml
+   azure.client.default.token_credential_type: "managed_identity"
+   ``` 
```

Comment:
```suggestion
``` 
```

Reply:
Thank you @Naarcha-AWS for fixing the syntax, and i have resolved the requested change as well. Cheers Chengwu